File: pidev/src/Entite/Medicament.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entite;

import java.util.List;

/**
 *
 * @author lafa
 */
public class Medicament {
     private int id_medicament;
    private String nom_medicament;
    private double prix;
    private String nom_labo;
    
    
    
    
    public Medicament(int id_medicament, String nom_medicament,double prix, String nom_labo){
        this.id_medicament=id_medicament;
        this.nom_medicament=nom_medicament;
        this.prix =prix;
        this.nom_labo=nom_labo;
    }

    public Medicament() {
        }

    public String getNom_labo() {
        return nom_labo;
    }

    public void setNom_labo(String nom_labo) {
        this.nom_labo = nom_labo;
    }

    public int getId_medicament() {
        return id_medicament;
    }

    public String getNom_medicament() {
        return nom_medicament;
    }

    public void setId_medicament(int id_medicament) {
        this.id_medicament = id_medicament;
    }

    public void setNom_medicament(String nom_medicament) {
        this.nom_medicament = nom_medicament;
    }

    public double getPrix() {
        return prix;
    }

    public void setPrix(double prix) {
        this.prix = prix;
    }
    

    @Override
    public String toString() {
        return "Servicesmedicament{" + "id_medicament=" + id_medicament + ", nom_medicament=" + nom_medicament + '}';
    }

   
    
}

File: pidev1/src/UserBundle/Entity/Medicament.php
<?php

namespace UserBundle\Entity;

/**
 * Medicament
 */
class Medicament
{
    /**
     * @var string
     */
    private $nomMedicament;

    /**
     * @var float
     */
    private $prix;

    /**
     * @var string
     */
    private $nomLabo;

    /**
     * @var string
     */
    private $usrrole;

    /**
     * @var integer
     */
    private $idMedicament;


    /**
     * Set nomMedicament
     *
     * @param string $nomMedicament
     *
     * @return Medicament
     */
    public function setNomMedicament($nomMedicament)
    {
        $this->nomMedicament = $nomMedicament;

        return $this;
    }

    /**
     * Get nomMedicament
     *
     * @return string
     */
    public function getNomMedicament()
    {
        return $this->nomMedicament;
    }

    /**
     * Set prix
     *
     * @param float $prix
     *
     * @return Medicament
     */
    public function setPrix($prix)
    {
        $this->prix = $prix;

        return $this;
    }

    /**
     * Get prix
     *
     * @return float
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * Set nomLabo
     *
     * @param string $nomLabo
     *
     * @return Medicament
     */
    public function setNomLabo($nomLabo)
    {
        $this->nomLabo = $nomLabo;

        return $this;
    }

    /**
     * Get nomLabo
     *
     * @return string
     */
    public function getNomLabo()
    {
        return $this->nomLabo;
    }

    /**
     * Set usrrole
     *
     * @param string $usrrole
     *
     * @return Medicament
     */
    public function setUsrrole($usrrole)
    {
        $this->usrrole = $usrrole;

        return $this;
    }

    /**
     * Get usrrole
     *
     * @return string
     */
    public function getUsrrole()
    {
        return $this->usrrole;
    }

    /**
     * Get idMedicament
     *
     * @return integer
     */
    public function getIdMedicament()
    {
        return $this->idMedicament;
    }
}

